pkgs/os-specific/linux/zfs/unstable.nix

```diff
@@ -17,7 +17,7 @@ callPackage ./generic.nix args {
   # check the release notes for compatible kernels
   kernelCompatible =
     if stdenv'.isx86_64 || removeLinuxDRM
-    then kernel.kernelOlder "6.9"
+    then kernel.kernelOlder "6.8"
```

Comment:
Upstream hasn't certified 6.8 on any of the branches. And understandably so, since 6.8 is not released. This is a downside of external modules. I didn't double check the version compatibility before merging the previous PR, and regret doing so. 

I appreciate there is a desire to run RC kernels, and this is named "unstable", but that doesn't mean we should offer a package that ignores upstream's recommendations on something as critical as a filesystem.

If there's something we need to change to allow easier overriding of zfs to enable unsupported configurations, let's improve that. But we need to make such configurations more opt-in than just choosing the "unstable" package.